### DISCUSSION TITLE
Why would vim change the colorscheme sourcecode?

### DISCUSSION THREAD

**Discussion by derekdilger** [vim/vim#14940] [Q&A]
Noob here. I just updated from vim 8.2 to 9. My vimrc file stayed the same. The colorscheme (torte) changed. What is the justification for this? Why not just add another colorscheme? I was accustomed to the 8.2 colors of torte. This feels like the smallest possible bait-and-switch. I realize it's possible for me to fix this-- that's great (but out of my way). But my question is about why a named scheme would ever be changed at all, given the assumption that people don't want something changed without their knowledge.

**Comment by derekdilger** [comment]
Apparently "They were made to work consistently across many types of terminals." Ok I guess.
I should have used :help version9 before making this post.

**Comment by lifepillar** [comment]
This has been discussed extensively in the past: see, for instance, https://github.com/vim/vim/issues/10449.

You know how to address this, but for the casual passer-by: see also `:help colorscheme-override`.
